Content.Server/LandMines/LandMineComponent.cs

```diff
@@ -1,6 +1,23 @@
-﻿namespace Content.Server.LandMines;
+namespace Content.Server.LandMines;
 
 [RegisterComponent]
 public sealed partial class LandMineComponent : Component
 {
+    /// <summary>
+    /// Defines whether landmine will explode when stepping on it or when stepping off it
+    /// True - StepTrigger, False - StepOffTrigger
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public bool ExplodeImmediately = false;
+
+    /// <summary>
+    /// Distance at which landmine beep sound can be heard by players
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    public readonly float TriggerAudioRange = 10f;
```

Comment:
no readonly + datafield

Reply:
Just use MaxDistance on the soundspecifier audioparams.